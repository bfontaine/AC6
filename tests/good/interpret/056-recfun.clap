def f (x) = if x < 0 then 42 else f (x-1)

val q = f(12)

