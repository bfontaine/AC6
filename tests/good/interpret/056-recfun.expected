:- f = <code>
:- q = 42
